python/source/robot_model/bind_model.cpp

```diff
@@ -52,6 +52,7 @@ void model(py::module_& m) {
 
   c.def("get_robot_name", &Model::get_robot_name, "Getter of the robot name.");
   c.def("set_robot_name", &Model::set_robot_name, "Setter of the robot name.", "robot_name"_a);
+  c.def("get_urdf", &Model::get_urdf, "Getter of the URDF XML string.");
   c.def("get_urdf_path", &Model::get_urdf_path, "Getter of the URDF path.");
```

Comment:
```suggestion
  c.def("get_urdf_path", &Model::get_urdf_path, "Getter of the URDF path if it was provided for construction instead of an XML string.");
```